src/storage-preview/azext_storage_preview/_help.py

```diff
@@ -207,6 +207,8 @@
     examples:
         - name: Create a storage blob directory in a storage container.
           text: az storage blob directory create -c MyContainer -d MyDirectoryPath --account-name MyStorageAccount
+        - name: Create a storage blob directory with permissions and umask.
+          text: az storage blob directory create -c MyContainer -d MyDirectoryPath --account-name MyStorageAccount --permissions rwxrwxrwx --umask 0000
```

Comment:
Is it dup? --permision and --umask. 

I think they do the same job but just differentiate in format.

Reply:
No. They will cooperate. Th final permission setting will be permissions & ^umask.

Reply:
emmm... Interesting...